publications-2018.bib

```diff
@@ -1485,4 +1485,183 @@ @article{Verner2018
+  year   = {2018},
+  url    = {https://cloudfront.escholarship.org/dist/prd/content/qt41f4p23x/qt41f4p23x.pdf},
+  School = {University of California, Davis}
+}
+
+
+@TechReport{Gulevich2018,
+  author = {Gulevich, D. R.},
+  title  = {{MiTMoJCo 1.2}},
+  year   = {2018},
+  url    = {https://www.researchgate.net/profile/D_Gulevich/publication/318380494_User_Guide_for_MiTMoJCo_12_Microscopic_Tunneling_Model_for_Josephson_Contacts/links/5ba4005d92851ca9ed1a09b1/User-Guide-for-MiTMoJCo-12-Microscopic-Tunneling-Model-for-Josephson-Contacts.pdf},
+}
+
+
+@Mastersthesis{Schuurmans2018,
+  author = {Schuurmans, L. F. J.},
+  title  = {{Numerical modelling of overriding plate deformation and slab rollback in the Western Mediterranean}},
+  School = {Utrecht University},
+  year   = {2018},
+  url    = {https://dspace.library.uu.nl/handle/1874/366408}
+}
+
+
+@incollection{Kronbichler2018,
+  doi = {10.1007/978-3-319-99654-7_7},
+  url = {https://doi.org/10.1007/978-3-319-99654-7_7},
+  year = {2018},
+  publisher = {Springer International Publishing},
+  pages = {89--110},
+  author = {Martin Kronbichler and Momme Allalen},
+  title = {Efficient High-Order Discontinuous Galerkin Finite Elements with Matrix-Free Implementations},
+  booktitle = {Progress in {IS}}
+}
+
+@article{Gassmller2018,
+  doi = {10.1029/2018gc007508},
+  url = {https://doi.org/10.1029/2018gc007508},
+  year = {2018},
+  month = sep,
+  publisher = {American Geophysical Union ({AGU})},
+  volume = {19},
+  number = {9},
+  pages = {3596--3604},
+  author = {Rene Gassm\"{o}ller and Harsha Lokavarapu and Eric Heien and Elbridge Gerry Puckett and Wolfgang Bangerth},
+  title = {Flexible and Scalable Particle-in-Cell Methods With Adaptive Mesh Refinement for Geodynamic Computations},
+  journal = {Geochemistry,  Geophysics,  Geosystems}
+}
+
+@incollection{Gantner2018,
+  doi = {10.1007/978-3-319-72456-0_18},
+  url = {https://doi.org/10.1007/978-3-319-72456-0_18},
+  year = {2018},
+  publisher = {Springer International Publishing},
+  pages = {373--405},
+  author = {Robert N. Gantner and Lukas Herrmann and Christoph Schwab},
+  title = {Multilevel {QMC} with Product Weights for Affine-Parametric,  Elliptic {PDEs}},
+  booktitle = {Contemporary Computational Mathematics - A Celebration of the 80th Birthday of Ian Sloan}
+}
+
+@article{Glerum2018,
+  doi = {10.5194/se-9-267-2018},
+  url = {https://doi.org/10.5194/se-9-267-2018},
+  year = {2018},
+  month = mar,
+  publisher = {Copernicus {GmbH}},
+  volume = {9},
+  number = {2},
+  pages = {267--294},
+  author = {Anne Glerum and Cedric Thieulot and Menno Fraters and Constantijn Blom and Wim Spakman},
+  title = {Nonlinear viscoplasticity in {\&}lt$\mathsemicolon$span style={\&}quot$\mathsemicolon${\&}quot$\mathsemicolon$ class={\&}quot$\mathsemicolon$text smallcaps{\&}quot$\mathsemicolon${\&}gt$\mathsemicolon${ASPECT}{\&}lt$\mathsemicolon$/span{\&}gt$\mathsemicolon$: benchmarking and applications to subduction},
```

Comment:
```suggestion
  title = {Nonlinear viscoplasticity in {ASPECT}: benchmarking and applications to subduction},
```

Reply:
Yes, I just found that a few seconds after you :-)

Reply:
That took a little while to find in the CI output :-)

Reply:
yep, a bit difficult, sadly.